Xext/xvdisp.c

```diff
@@ -841,7 +841,9 @@ ProcXvQueryImageAttributes(ClientPtr client)
         swapl(&rep.data_size);
         swaps(&rep.width);
         swaps(&rep.height);
-        SwapLongs((CARD32 *) offsets, rep.length);
+        /* needed here, because ddQueryImageAttributes() directly wrote into
+           our rpcbuf area */
+        SwapLongs((CARD32 *) offsets, x_rpcbuf_wsize_units(&rpcbuf));
```

Comment:
If `rep.length` was already calculated, why not reuse it, like it was done in the old version? Not a big deal though.